docs/contribution-guide/2-creating-pr.md

```diff
@@ -0,0 +1,35 @@
+---
+date: 2024-05-10
+category: document
+author: Revanth Rao Thopucherla
+order: 2
+
+---
+
+# Creating a Pull Request
+
+1. **Open the GitHub repository**: Navigate to the [product-docs repository](https://github.com/jangala-dev/product-docs) on GitHub.
+![alt text](images/github-code-tab.png)
+
+2. **Create a new branch**: Click on the `Code` tab and then click on the `Branch: main` dropdown. Enter a new branch name and click `Create branch`.
+![alt text](images/new-branch.png)
+
+3. **Navigate to the docs folder**: Navigate to the [`docs`](https://github.com/jangala-dev/product-docs/tree/main/docs)  folder in the repository to find the documentation files you want to edit.
+4. **Make changes**: Make your changes to the documentation using Github web Interface by clicking edit icon on the top right as shown in the below screenshot or using Local IDE. All the files are written in markdown format. You can refer to the [Markdown Guide](https://guides.github.com/features/mastering-markdown/) for help with formatting.
+![alt text](images/edit-icon.png)
+5. **Commit changes**: Once you've made your changes, commit them to your branch.
+![alt text](images/commit-changes.png)
+6. **Open a pull request**: Navigate to the `Pull requests` tab and click on `New pull request`. Select the branch you made changes to and the `main` branch. Click `Create pull request`.
+![alt text](images/create-pull-request.png)
+![alt text](images/branches.png)
+7. **Describe your changes**: Provide a title and description for your pull request. Click `Create pull request` to submit it.
+8. **Review and merge**: Assign the relevant person to review. Your pull request will be reviewed by the maintainers. Once approved, it will be merged into the main branch.
+9. **Delete your branch**: After your pull request is merged, you can delete your branch.
+
+## Video Walkthrough (Optional)
+
+<video src="/contributing-video.mp4"  height="240" controls></video>
+
+## End Credits
```

Comment:
I would call this conclusion

Reply:
yeah co pilot suggested conclusion but i went for end credits ;) . Done